crates/core/tedge_api/src/entity_store.rs

```diff
@@ -12,19 +12,19 @@ use std::collections::HashMap;
 use mqtt_channel::Message;
 use mqtt_channel::Topic;
 
-/// Represents an "MQTT entity identifier" portion of the MQTT topic
+/// Represents an "Entity MQTT address" portion of the MQTT topic
 ///
 /// Example:
-/// - topic: `te/device/dev1/service/myservice/m//my_measurement
+/// - topic: `te/device/dev1/service/myservice/m//my_measurement`
 /// - entity id: `device/dev1/service/myservice`
 ///
 /// <https://thin-edge.github.io/thin-edge.io/next/references/mqtt-api/#group-identifier>
-type EntityId = String;
+type EntityMqttAddress = String;
```

Comment:
@Bravo555, @reubenmiller, @albinsuresh 

I'm not fully happy with this proposal for a new name:

Pros:
- `EntityId` is definitely confusing, knowing that *device id* is already used with a [different meaning](https://thin-edge.github.io/thin-edge.io/next/references/mappers/c8y-mapper/#child-device-with-explicit-id): (the main device id is the CN of its certificate while its *entity identifier* is `device/main//`.

Cons:
- The specifications uses the term [*entity identifier*](https://thin-edge.github.io/thin-edge.io/next/references/mqtt-api/#group-identifier).
- The *address* suffix of `EntityMqttAddress` might be confused with some kind of network address.
- This identifier is used only for MQTT topics. Hence, I would expect the term *topic* rather than *address*

What about `EntityTopicId` or `EntityMqttTopicId` ?

## Alternative

Another way to solve the issue is t:
- keep the terms `EntityId` and entity identifier, when speaking about topics.
- use the terms `EntityName` and entity name, when speaking about the CommonName (CN) of these devices.

## Important

In any case, the terms used in the code base must be the same as those used in the docs.

Reply:
We agreed to use the name `EntityTopicId` for entity identifiers in the new topic scheme.